src/test/java/org/cactoos/iterator/SyncedTest.java

```diff
@@ -38,6 +38,17 @@
  * @checkstyle JavadocMethodCheck (500 lines)
  * @checkstyle MagicNumberCheck (500 lines)
  * @checkstyle TodoCommentCheck (500 lines)
+ * @todo #1115:30min Continue speeding up the `mvn test` goal until it
+ *  executes in less than 10 seconds, or as fast as reasonably possible.
+ *  A good solution would be reducing the number of tries to at least 10
+ *  in the two tests below with the for loop. But in order to do so, the
+ *  robustness of RunsInThreads must be improved: after some manual testing
+ *  to evaluate actual overlaps of threads (see code details in
+ *  https://www.yegor256.com/2018/03/27/how-to-test-thread-safety.html),
+ *  it seems most of the tries results in 0 overlapping thread. This is
+ *  why so much tries are needed in the for loop for the test to be sound.
+ *  If RunsInThreads was robust enough to guarantee overlapping, only one try
```

Comment:
@victornoel I'm not sure what "robustness" you're referring to. Can you explain?

Reply:
@llorllale yes, it's not clear: the purpose of `RunsInThreads` is to test concurrent execution. The problem is that sometimes, it does not run the task concurrently in various thread: the threads do not overlaps (see https://www.yegor256.com/2018/03/27/how-to-test-thread-safety.html).
So it is not robust w.r.t. its expected behaviour. Hence the sentence "If RunsInThreads was robust enough to guarantee overlapping" which implicitly suppose that the correct behaviour of `RunsInThreads` os to have overlapping threads.

Reply:
@victornoel you'd have to open a ticket in cactoos-matchers so we can triage this there. 

However, warning: I don't see how `RunsInThread` can be improved to increase "overlap". We literally have a `CountDownLatch` for this.

Reply:
@llorllale it is here: https://github.com/llorllale/cactoos-matchers/issues/127. If it is accepted I will update this todo to reference it and propose to reduce the number of loops when it is solved. WDYT?

Reply:
@victornoel answered there

Reply:
@llorllale I've updated the todo to reflect the conclusion we arrived to